src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import { Alert } from "react-bootstrap";
+
+interface Props {
+  message: string;
+}
+export default function AlertDismissible({ message }: Props) {
+  const [shown, setShow] = useState(true);
```

Comment:
ah sorry could this also be `setShown` instead of `setShow`?

Reply:
done
!